tests/test_webrtc_util.py

```diff
@@ -1,17 +1,84 @@
+        "c=IN IP4 0.0.0.0\r\n"
+        "a=recvonly\r\n"
+        "m=application 9 UDP/DTLS/SCTP webrtc-datachannel\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+        "a=sendrecv\r\n"
+    )
+    answer_sdp = (
+        "v=0\r\n"
+        "o=- 0 2 IN IP4 127.0.0.1\r\n"
+        "m=audio 19305 UDP/TLS/RTP/SAVPF 109\r\n"
+        "c=IN IP4 74.125.247.118\r\n"
+        "a=rtcp:9 IN IP4 0.0.0.0\r\n"
+        "a=candidate: 1 udp 2113939711 2001:4860:4864:4::118 19305 typ host generation 0\r\n"
+        "a=candidate: 1 tcp 2113939710 2001:4860:4864:4::118 19305 typ host tcptype passive generation 0\r\n"
+        "a=candidate: 1 ssltcp 2113939709 2001:4860:4864:4::118 443 typ host generation 0\r\n"
+        "a=candidate: 1 udp 2113932031 74.125.247.118 19305 typ host generation 0\r\n"
+        "a=candidate: 1 tcp 2113932030 74.125.247.118 19305 typ host tcptype passive generation 0\r\n"
+        "a=candidate: 1 ssltcp 2113932029 74.125.247.118 443 typ host generation 0\r\n"
+        "a=sendrecv\r\n"
+        "m=video 9 UDP/TLS/RTP/SAVPF 126 127\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+        "a=rtcp:9 IN IP4 0.0.0.0\r\n"
+        "a=sendrecv\r\n"
+        "m=application 9 DTLS/SCTP 5000\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+    )
+    expected_answer_sdp = (
+        "v=0\r\n"
+        "o=- 0 2 IN IP4 127.0.0.1\r\n"
+        "m=audio 19305 UDP/TLS/RTP/SAVPF 109\r\n"
+        "c=IN IP4 74.125.247.118\r\n"
+        "a=rtcp:9 IN IP4 0.0.0.0\r\n"
+        "a=candidate:1 1 udp 2113939711 2001:4860:4864:4::118 19305 typ host generation 0\r\n"
+        "a=candidate:2 1 tcp 2113939710 2001:4860:4864:4::118 19305 typ host tcptype passive generation 0\r\n"
+        "a=candidate:3 1 ssltcp 2113939709 2001:4860:4864:4::118 443 typ host generation 0\r\n"
+        "a=candidate:4 1 udp 2113932031 74.125.247.118 19305 typ host generation 0\r\n"
+        "a=candidate:5 1 tcp 2113932030 74.125.247.118 19305 typ host tcptype passive generation 0\r\n"
+        "a=candidate:6 1 ssltcp 2113932029 74.125.247.118 443 typ host generation 0\r\n"
+        "a=sendonly\r\n"
+        "m=video 9 UDP/TLS/RTP/SAVPF 126 127\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+        "a=rtcp:9 IN IP4 0.0.0.0\r\n"
+        "a=sendonly\r\n"
+        "m=application 9 DTLS/SCTP 5000\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+    )
+    chrome_offer_sdp = (
+        "v=0\r\n"
+        "o=- 6714414228100263102 2 IN IP4 127.0.0.1\r\n"
+        "m=audio 9 UDP/TLS/RTP/SAVPF 111 63 9 0 8 13 110 126\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+        "a=recvonly\r\n"
+        "m=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 100 101 35 36 37 38 102 103 104 105 106 107 108 109 127 125 39 40 41 42 43 44 45 46 47 48 112 113 114 115 116 117 118 49\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+        "a=recvonly\r\n"
+        "m=application 9 UDP/DTLS/SCTP webrtc-datachannel\r\n"
+        "c=IN IP4 0.0.0.0\r\n"
+    )
+    fixed_sdp = fix_mozilla_sdp_answer(firefox_offer_sdp, answer_sdp)
+    assert expected_answer_sdp == fixed_sdp
```

Comment:
The style in this repo is with the expected result on the right. Fix here and below please.

```suggestion
    assert fixed_sdp == expected_answer_sdp
```